src/pybind/mgr/dashboard/frontend/src/app/ceph/cluster/services/service-form/service-form.component.html

```diff
@@ -508,7 +508,7 @@
           </div>
 
           <!-- ssl_key -->
-          <div *ngIf="serviceForm.controls.ssl.value"
+          <div *ngIf="serviceForm.controls.ssl.value && ['iscsi', 'ingress'].includes(serviceForm.controls.service_type.value)"
```

Comment:
Just curious, why not something like `serviceForm.controls.service_type.value !== 'rgw'` or something like that? I mean check whether the type is not rgw and then show the field?

Reply:
Oh yes..I was looking for the same..thanks @nizamial09 . Yeah it makes more sense to exclude here!

Reply:
Done.